pkgs/os-specific/linux/mstflint_access/default.nix

```diff
@@ -29,7 +29,11 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
-    install -D ${pname}.ko $out/lib/modules/${kernel.modDirVersion}/extra/${pname}.ko
+    make -C ${kernel.dev}/lib/modules/${kernel.modDirVersion}/build \
```

Comment:
Can't we use `installFlags` for this?

Reply:
I've changed this to use `installFlags` and `installTargets` now.